include/zim/zim.h

```diff
@@ -33,6 +33,11 @@
 #define DEPRECATED
 #endif
 
+#if (defined _WIN32 || defined __CYGWIN__) && defined ENABLE_EXPORT_DLL
```

Comment:
Let's rename the `ENABLE_EXPORT_DLL` macro to `LIBZIM_EXPORT_DLL`. Otherwise, we may run into build problems when `libzim` is used as a dependency in a project that uses `ENABLE_EXPORT_DLL` macro for exporting its own dll.